packages/api-client-core/src/operationBuilders.ts

```diff
@@ -206,25 +206,32 @@ export const backgroundActionResultOperation = <Action extends AnyActionFunction
   options?: Options
 ) => {
   let fields: FieldSelection = {};
-  let operationName = action.operationName;
   let resultType: string;
 
-  if (action.isBulk) {
-    operationName = action.operationName.replace(/^bulk/, "").replace(/s$/, "");
+  const backgroundAction = action.isBulk && action.singleAction ? action.singleAction : action;
```

Comment:
before we make changes in core, `action.singleAction` will always be `undefined` and this will be equivalent to the current case